state-chain/pallets/cf-ingress-egress/src/benchmarking.rs

```diff
@@ -145,6 +159,26 @@ mod benchmarks {
 		);
 	}
 
+	#[benchmark]
+	fn remove_prewitnessed_deposits(n: Linear<1, 255>) {
+		for i in 0..n {
+			PrewitnessedDeposits::<T, I>::insert(
+				0,
+				i as u64,
+				PrewitnessedDeposit {
+					asset: BenchmarkValue::benchmark_value(),
+					amount: BenchmarkValue::benchmark_value(),
+					deposit_address: BenchmarkValue::benchmark_value(),
+				},
+			);
+		}
+
+		#[block]
+		{
+			let _old_deposits = PrewitnessedDeposits::<T, I>::clear_prefix(0, u32::MAX, None);
```

Comment:
Better to pull the benchmarked part into a function rather than effectively duping this here and where it's actually called in the running code

Reply:
and then we can add a simple check that they were removed too